options.go

```diff
@@ -0,0 +1,85 @@
+package jrpc
+
+import (
+	"net/http"
+	"time"
+)
+
+// Option func type
+type Option func(s *Server)
+
+// Auth sets basic auth credentials, required
+func Auth(user, password string) Option {
+	return func(s *Server) {
+		s.authUser = user
+		s.authPasswd = password
+	}
+}
+
+// AppName sets custom application name for server signature, optional
+func AppName(name string) Option {
```

Comment:
Probably it makes sense to add author the same way

Reply:
What's default value should be define for `author` field, empty or '_umputun_' ?

Reply:
empty should be fine